app/services/barcode_outputter.rb

```diff
@@ -0,0 +1,15 @@
+require 'barby'
+require 'barby/barcode/code_128'
+require 'barby/outputter/png_outputter'
+
+class BarcodeOutputter
```

Comment:
maybe drop a comment that this is specifically one format of barcode meant for USPS? The name of the class makes it sound like it's for any barcode, but this has a specific audience

Reply:
> maybe drop a comment that this is specifically one format of barcode meant for USPS? The name of the class makes it sound like it's for any barcode, but this has a specific audience

That's true, and I did pause and reflect on the fact this can only really support the Code128C-type barcodes.

In general, my expectation for this class is that it _could_ support barcodes intended for other purposes, and that we could seek to enhance the implementation if / when we need that. And that the purpose of the class is more on standardizing the options / configuration / `require`s involved with using the Barby gem.

I can include a comment with something to that effect if it's useful, though also want to avoid running another build 😅 (Could always admin-bypass I suppose)

Reply:
Added comment in 2f5015c45